.gitignore

```diff
@@ -3,6 +3,7 @@
 /build/wrapper_table.json
 /build/doc_env/
 /build/py_env/
+/build/py27_env/
```

Comment:
Can you try removing this directory in your local checkout and seeing if it comes back? I believe this was renamed in the Makefiles to py_env recently, so you should be able to remove this.